Cargo.toml

```diff
@@ -13,6 +13,11 @@ members = [
 [workspace.package]
 authors = ["Bitcoin Dev Kit Developers"]
 
+[workspace.dependencies]
+tokio = "=1.38.1"
+reqwest = "=0.12.4"
+socket2 = "=0.5.10"
```

Comment:
I don't think this is required, it should be already covered in CI by the `ci/pin-msrv.sh` used in CI.